docs/quick-install.md

```diff
@@ -1,82 +1,111 @@
+If the download succeeds, you don't see any output, but you do see a new command prompt.
 
-From the **Setup** tab, copy and paste the following code into your command line or terminal:
+3. When the download finishes, run the install script in a command line window. Copy the following command and paste it into the window of `terminal` or a similar app.
+`sudo sh /tmp/signalfx-agent.sh --realm YOUR_SIGNALFX_REALM -- YOUR_SIGNALFX_API_TOKEN`
 
+When this command finishes, it displays the following:
 
-```sh
-curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
-sudo sh /tmp/signalfx-agent.sh --realm YOUR_SIGNALFX_REALM -- YOUR_SIGNALFX_API_TOKEN
-```
+`The SignalFx Agent has been successfully installed.`
 
+`Make sure that your system's time is relatively accurate or else datapoints may not be accepted.`
 
+`The agent's main configuration file is located at /etc/signalfx/agent.yaml.`
 
-#### Windows
+4. (Optional) If you want to override the default user and group names, review your [deployment's README file](https://github.com/signalfx/signalfx-agent/tree/master/deployments) and locate the option to set the user/group ownership for the signalfx-agent service.
 
-Note: This content appears on a SignalFx documentation page and on the **Setup** tab of the Smart Agent tile in the SignalFx UI. The following code to install the current version works only if you are viewing these instructions on the **Setup** tab.  
+5. If your installation succeeds, proceed to the section **Verify Your Installation**. Otherwise, see the section **Troubleshoot Your Installation**.
 
-From the **Setup** tab, copy and paste the following code into your PowerShell window:
+### Install SignalFx Smart Agent on Windows
 
+1. Run Windows PowerShell.
+
+2. To configure Windows execution policy, download the Smart Agent install script, and run it, copy the following commands and paste them into the Windows PowerShell window:
 
 ```sh
-& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"; ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
+& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString(‘https://dl.signalfx.com/signalfx-agent.ps1’)); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"; ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(“. {$script} $(&{$args} @params)”))}
```

Comment:
Are the changes to the single and double quotes in the command necessary? Seems like it could cause issues if a user copies and pastes this command.

Reply:
We received a ticket that this was a bug, so we made the changes listed here: https://signalfuse.atlassian.net/browse/DOCS-1432

Reply:
Thanks @galpizarsfx, didn't realize the quotes were getting rendered differently in the UI.